apps/src/acemode/mode-javascript_codeorg.js

```diff
@@ -40,6 +40,8 @@ exports.defineForAce = function (dropletConfig, unusedConfig, dropletEditor) {
         var worker = new WorkerClient(["ace"], "ace/mode/javascript_worker", "JavaScriptWorker");
         worker.attachToDocument(session.getDocument());
         var newOptions = {
+          '-W041': false,
```

Comment:
(just curious) what does '-W041' do

Reply:
Good question! I believe it's an alias for the `eqnull` null rule, but I was having trouble turning it off the other way.